### PR TITLE
Topic ui

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: api/controllers/TagController.js [hunk 1]
```diff
@@ -21,27 +22,30 @@ module.exports = {
...
-      ))
+      return Promise.join(
+        tagFollowers(res.locals.community, tag),
+        CommunityTag
+        .where({community_id: res.locals.community.id, tag_id: tag.id})
+        .fetch({withRelated: [
+          'owner',
+          {'community.tagFollows': q => q.where({
+            'tag_follows.tag_id': tag.id,
+            'tag_follows.user_id': req.session.userId
```
Comment (levity) [inline api/controllers/TagController.js]: couldn't you just drop this user_id where clause and get the followers that way?

Comment (robbiecarlton) [inline api/controllers/TagController.js]: that would give me the ids, but then I still need the models

**Comment by robbiecarlton** [comment]
how's this?

**Pull request merged**